yjit/src/core.rs

```diff
@@ -919,7 +924,7 @@ impl Context {
     {
         unsafe {
             if CTX_CACHE == None {
-                CTX_CACHE = Some( [(Context::default(), 0); CTX_CACHE_SIZE] );
+                CTX_CACHE = CtxCache::default();
```

Comment:
Since `CtxCache` is an `Option`, is this `if CTX_CACHE == None { CTX_CACHE = None; }`?

Reply:
Aah I see what you mean 😆